docs/getting-started.md

```diff
@@ -116,7 +116,7 @@ curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version
 tar.exe xvf .\containerd-windows-amd64.tar.gz
 
 # Copy and configure
-Copy-Item -Path ".\bin\*" -Destination "$Env:ProgramFiles\containerd" -Recurse -Force
+Copy-Item -Path .\bin\* -Destination (New-Item -Type Directory $Env:ProgramFiles\containerd) -Recurse -Force
```

Comment:
I think `new-item` will fail if the path already exists. Maybe make that a separate command that checks if the path exists?

Reply:
It does, there are two other solutions:
- `New-Item -Type Directory -Force ...`
- `New-Item -Type Directory -ErrorAction Ignore ...`
I think force would be preferred over ignoring errors though

Reply:
Sure, let me go with `-Force`, to also help with re-install or upgrade scenarios.